indico/MaKaC/webinterface/tpls/RoomBookingSearchRoomsResults.tpl

```diff
@@ -15,62 +15,62 @@
-                              <td width="65%" class="dataCaptionFormat">${ _('Actions') }</td>
-                            </tr>
-                            <tr>
-                              <td class="titleCellTD" colspan="10" style="height: 0px">&nbsp;</td>
-                            </tr>
-                            % for room in rooms:
-                              <% details_url = url_for('rooms.roomBooking-roomDetails', room) %>
-                              <% booking_url = url_for('rooms.room_book', room) %>
-                              <% modification_url = url_for('rooms_admin.modify_room', room) %>
-                              <% on_click_details_url = 'onclick="window.location=\'{}\'"'.format(details_url) %>
-                              % if mapAvailable:
-                                <% show_on_map = url_for('rooms.roomBooking-mapOfRooms', roomID=room.id) %>
-                              % endif
-                              <tr style="height: 60px" id="${ room.id }" class="resvHover">
-                                <td ${ on_click_details_url }>
-                                  <img src="${ room.small_photo_url }" />
-                                </td>
-                                <td ${ on_click_details_url }>
-                                  ${ room.building }-${ room.floor }-${ room.number }
-                                  % if room.has_special_name:
-                                    <small>(${ room.name })</small>
-                                  % endif
-                                </td>
-                                <td ${ on_click_details_url } align='center'>
-                                  ${ room.capacity if room.capacity is not None else _('n/a') }
-                                </td>
-                                <td ${ on_click_details_url }>
-                                  ${ room.site }
-                                </td>
-                                <td>
-                                  <a href="${ details_url }">${ _('view') }</a><br/>
-                                  % if room.can_be_booked(user):
-                                    <a href="${ booking_url }">${ _('book') }</a><br/>
-                                  % elif room.can_be_prebooked(user):
-                                    <a href="${ booking_url }">${ _('PRE-book') }</a><br/>
-                                  % endif
-                                  % if room.can_be_modified(user):
-                                    <a href="${ modification_url }"> ${ _('modify') }</a><br/>
-                                  % endif
-                                  % if mapAvailable:
-                                    <a href="${ show_on_map }"> ${ _('show on map') }</a><br/>
-                                  % endif
-                                </td>
+                          <table class="resvTable tablesorter" style="width: 100%">
```

Comment:
since the indentation for the whole thing changed due to you adding thead/tbody, why not use the opportunity to indent with 4 spaces?